[
  {
    "type": "value_error",
    "loc": [
      "contributor",
      52,
      "Person"
    ],
    "msg": "Value error, Contact person must have an email address.",
    "input": {
      "name": "Tye, Kay M. ",
      "roleName": [
        "dcite:Conceptualization",
        "dcite:ContactPerson",
        "dcite:Supervision",
        "dcite:Author"
      ],
      "schemaKey": "Person",
      "includeInCitation": true
    },
    "ctx": {
      "error": "Contact person must have an email address."
    },
    "url": "https://errors.pydantic.dev/2.10/v/value_error"
  }
]